vault/mount.go

```diff
@@ -1603,7 +1605,8 @@ func (c *Core) setupMounts(ctx context.Context) error {
 					view.setReadOnlyErr(origReadOnlyErr)
 				}
 
-				err := backend.Initialize(ctx, &logical.InitializationRequest{Storage: view})
+				nsActiveContext := namespace.ContextWithNamespace(c.activeContext, localEntry.Namespace())
```

Comment:
The `ctx` passed into `setupMounts` is the core's `activeContext`. I can change this back to `ctx`. I figured `c.activeContext` was more explicit and matches what's provided to `Initialize` in `mountInternal`.